Automattic/simplenote-electron
# Save export zip without using `fs.writeFile`

This removes the need to use `fs.writeFile` for the zip export, so we can keep the export functionality in a non-Electron web app.

### To test

This doesn't actually implement an Export button in the UI yet, so here we can just make sure that the Export Notes menu command still works as before. Or, if you want to test in a browser, add a `exportZipArchive('test.zip')` call in the `componentDidMount()` of `app.jsx` 🙂 


Note for code review: It will be a lot easier to follow the commits in order, rather than the full diff.
